diff --git a/tests/Unit/Entity/DinosaurTest.php b/tests/Unit/Entity/DinosaurTest.php
index 52532a2..ee588d7 100644
--- a/tests/Unit/Entity/DinosaurTest.php
+++ b/tests/Unit/Entity/DinosaurTest.php
@@ -36,7 +36,7 @@ class DinosaurTest extends TestCase
         self::assertSame($specification, $dino->getSpecification());
     }
 
-    public function getDinoLengthSpecification()
+    public function getDinoLengthSpecification(): \Generator
     {
         yield '15 Meter Large Dino' => [15, 'Large'];
         yield '7 Meter Medium Dino' => [7, 'Medium'];
