diff --git a/tests/Unit/Entity/DinosaurTest.php b/tests/Unit/Entity/DinosaurTest.php
index 37f52de..542c42d 100644
--- a/tests/Unit/Entity/DinosaurTest.php
+++ b/tests/Unit/Entity/DinosaurTest.php
@@ -40,6 +40,6 @@ class DinosaurTest extends TestCase
     {
         yield '15 Meter Large Dino' => [15, 'Large'];
         yield '7 Meter Medium Dino' => [7, 'Medium'];
-        yield '4 Meter Small Dino' => [4, 'Small'];
+        yield '4 Meter Small Dino' => [15, 'Small'];
     }
 }
