diff --git a/tests/Integration/Repository/LockDownRepositoryTest.php b/tests/Integration/Repository/LockDownRepositoryTest.php
index ac0dca8..74d51f5 100644
--- a/tests/Integration/Repository/LockDownRepositoryTest.php
+++ b/tests/Integration/Repository/LockDownRepositoryTest.php
@@ -15,4 +15,9 @@ class LockDownRepositoryTest extends KernelTestCase
         assert($lockDownRepository instanceof LockDownRepository);
         $this->assertFalse($lockDownRepository->isInLockDown());
     }
+
+    public function testIsInLockDownReturnsTrueIfMostRecentLockDownIsActive()
+    {
+        self::bootKernel();
+    }
 }
