diff --git a/tests/Unit/Entity/DinosaurTest.php b/tests/Unit/Entity/DinosaurTest.php
index 78505ee..fa5a6be 100644
--- a/tests/Unit/Entity/DinosaurTest.php
+++ b/tests/Unit/Entity/DinosaurTest.php
@@ -34,8 +34,8 @@ class DinosaurTest extends TestCase
 
     public function sizeDescriptionProvider()
     {
-        yield [10, 'Large'];
-        yield [5, 'Medium'];
-        yield [4, 'Small'];
+        yield '10 Meter Large Dino' => [10, 'Large'];
+        yield '5 Meter Medium Dino' => [5, 'Medium'];
+        yield '4 Meter Small Dino' => [4, 'Small'];
     }
 }
