diff --git a/tests/Unit/Entity/DinosaurTest.php b/tests/Unit/Entity/DinosaurTest.php
index fa3f283..7cc36c8 100644
--- a/tests/Unit/Entity/DinosaurTest.php
+++ b/tests/Unit/Entity/DinosaurTest.php
@@ -22,6 +22,9 @@ class DinosaurTest extends TestCase
         self::assertSame('Paddock A', $dino->getEnclosure());
     }
 
+    /**
+     * @dataProvider getDinoSizeDescription
+     */
     public function testDinoOver10MetersIsLarge(int $length, string $expectedSize): void
     {
         $dino = new Dinosaur(name: 'Big Eaty', length: $length);
