diff --git a/tests/Unit/Entity/DinosaurTest.php b/tests/Unit/Entity/DinosaurTest.php
index 674e79e..1068ef6 100644
--- a/tests/Unit/Entity/DinosaurTest.php
+++ b/tests/Unit/Entity/DinosaurTest.php
@@ -22,24 +22,17 @@ class DinosaurTest extends TestCase
         self::assertSame('Paddock A', $dino->getEnclosure());
     }
 
-    public function testDino10MetersOrGreaterIsLarge(): void
+    public function testDino10MetersOrGreaterIsLarge(int $length, string $expectedSize): void
     {
-        $dino = new Dinosaur(name: 'Big Eaty', length: 10);
+        $dino = new Dinosaur(name: 'Big Eaty', length: $length);
 
-        self::assertSame('Large', $dino->getSizeDescription(), 'This is supposed to be a large Dinosaur');
+        self::assertSame($expectedSize, $dino->getSizeDescription(), 'This is supposed to be a large Dinosaur');
     }
 
-    public function testDinoBetween5And9MetersIsMedium(): void
+    public function sizeDescriptionProvider()
     {
-        $dino = new Dinosaur(name: 'Big Eaty', length: 5);
-
-        self::assertSame('Medium', $dino->getSizeDescription(), 'This is supposed to be a medium Dinosaur');
-    }
-
-    public function testDinoUnder5MetersIsSmall(): void
-    {
-        $dino = new Dinosaur(name: 'Big Eaty', length: 4);
-
-        self::assertSame('Small', $dino->getSizeDescription(), 'This is supposed to be a small Dinosaur');
+        yield [10, 'Large'];
+        yield [5, 'Medium'];
+        yield [4, 'Small'];
     }
 }
