diff --git a/tests/Integration/Repository/LockDownRepositoryTest.php b/tests/Integration/Repository/LockDownRepositoryTest.php
index 788bad4..193c088 100644
--- a/tests/Integration/Repository/LockDownRepositoryTest.php
+++ b/tests/Integration/Repository/LockDownRepositoryTest.php
@@ -29,7 +29,7 @@ class LockDownRepositoryTest extends KernelTestCase
         $entityManager->persist($lockDown);
         $entityManager->flush();
 
-        dd($lockDown->getId());
+        $this->assertTrue($this->getLockDownRepository()->isInLockDown());
     }
 
     private function getLockDownRepository(): LockDownRepository
