diff --git a/tests/Unit/Entity/DinosaurTest.php b/tests/Unit/Entity/DinosaurTest.php
index eca3588..b67ad61 100644
--- a/tests/Unit/Entity/DinosaurTest.php
+++ b/tests/Unit/Entity/DinosaurTest.php
@@ -22,6 +22,9 @@ class DinosaurTest extends TestCase
         self::assertSame('Paddock A', $dino->getEnclosure());
     }
 
+    /**
+     * @dataProvider getDinoLengthSpecification
+     */
     public function testDinoOver15MetersIsLarge(int $length, string $specification): void
     {
         $dino = new Dinosaur(
