diff --git a/tests/Unit/Entity/DinosaurTest.php b/tests/Unit/Entity/DinosaurTest.php
index 1068ef6..78505ee 100644
--- a/tests/Unit/Entity/DinosaurTest.php
+++ b/tests/Unit/Entity/DinosaurTest.php
@@ -22,6 +22,9 @@ class DinosaurTest extends TestCase
         self::assertSame('Paddock A', $dino->getEnclosure());
     }
 
+    /**
+     * @dataProvider sizeDescriptionProvider
+     */
     public function testDino10MetersOrGreaterIsLarge(int $length, string $expectedSize): void
     {
         $dino = new Dinosaur(name: 'Big Eaty', length: $length);
