diff --git a/tests/Unit/Entity/DinosaurTest.php b/tests/Unit/Entity/DinosaurTest.php
index 10756c6..bbb5901 100644
--- a/tests/Unit/Entity/DinosaurTest.php
+++ b/tests/Unit/Entity/DinosaurTest.php
@@ -38,4 +38,11 @@ class DinosaurTest extends TestCase
         yield '5 Meter Medium Dino' => [5, 'Medium'];
         yield '4 Meter Small Dino' => [4, 'Small'];
     }
+
+    public function testIsAcceptingVisitorsByDefault(): void
+    {
+        $dino = new Dinosaur('Dennis');
+
+        self::assertTrue($dino->isAcceptingVisitors());
+    }
 }
