diff --git a/tests/Unit/Entity/DinosaurTest.php b/tests/Unit/Entity/DinosaurTest.php
index 94e2821..470b608 100644
--- a/tests/Unit/Entity/DinosaurTest.php
+++ b/tests/Unit/Entity/DinosaurTest.php
@@ -18,7 +18,7 @@ class DinosaurTest extends TestCase
 
         self::assertSame('Big Eaty', $dino->getName());
         self::assertSame('Tyrannosaurus', $dino->getGenus());
-        self::assertSame(15, $dino->getLength());
+        self::assertSame(5, $dino->getLength());
         self::assertSame('Paddock A', $dino->getEnclosure());
     }
 }
