diff --git a/tests/Unit/Entity/DinosaurTest.php b/tests/Unit/Entity/DinosaurTest.php
index bbb5901..152e546 100644
--- a/tests/Unit/Entity/DinosaurTest.php
+++ b/tests/Unit/Entity/DinosaurTest.php
@@ -45,4 +45,13 @@ class DinosaurTest extends TestCase
 
         self::assertTrue($dino->isAcceptingVisitors());
     }
+
+    public function testIsNotAcceptingVisitorsIfSick(): void
+    {
+        $dino = new Dinosaur('Bumpy');
+
+        $dino->setHealth('Sick');
+
+        self::assertFalse($dino->isAcceptingVisitors());
+    }
 }
