diff --git a/src/Service/LockDownHelper.php b/src/Service/LockDownHelper.php
index acfdca2..e66ad1e 100644
--- a/src/Service/LockDownHelper.php
+++ b/src/Service/LockDownHelper.php
@@ -6,13 +6,16 @@ use App\Entity\LockDown;
 use App\Enum\LockDownStatus;
 use App\Repository\LockDownRepository;
 use Doctrine\ORM\EntityManagerInterface;
+use Symfony\Component\Mailer\MailerInterface;
+use Symfony\Component\Mime\Email;
 
 class LockDownHelper
 {
     public function __construct(
         private LockDownRepository $lockDownRepository,
         private EntityManagerInterface $entityManager,
-        private GithubService $githubService
+        private GithubService $githubService,
+        private MailerInterface $mailer
     )
     {
     }
@@ -37,6 +40,18 @@ class LockDownHelper
         $lockDown->setReason('Dino escaped... NOT good...');
         $this->entityManager->persist($lockDown);
         $this->entityManager->flush();
-        // send an email with subject like "RUUUUUUNNNNNN!!!!"
+        $this->sendEmailAlert();
+    }
+
+    private function sendEmailAlert(): void
+    {
+        $email = (new Email())
+            ->from('bob@dinotopia.com')
+            ->to('staff@dinotopia.com')
+            ->subject('PARK LOCKDOWN')
+            ->text('RUUUUUUNNNNNN!!!!')
+        ;
+
+        $this->mailer->send($email);
     }
 }
