diff --git a/tests/Unit/Service/GithubServiceTest.php b/tests/Unit/Service/GithubServiceTest.php
index cb77e18..f89c10c 100644
--- a/tests/Unit/Service/GithubServiceTest.php
+++ b/tests/Unit/Service/GithubServiceTest.php
@@ -109,6 +109,10 @@ class GithubServiceTest extends TestCase
                     'title' => 'Dennis',
                     'labels' => [['name' => 'Status: Hungry']],
                 ],
+                [
+                    'title' => 'Big Eaty',
+                    'labels' => [['name' => 'Escaped!']],
+                ],
                 [
                     'title' => 'Bumpy',
                     'labels' => [['name' => 'Status: Drowsy']],
@@ -125,6 +129,7 @@ class GithubServiceTest extends TestCase
         $service = new GithubService($mockHttpClient, $this->createMock(LoggerInterface::class));
 
         $this->expectException(\RuntimeException::class);
+        $this->expectExceptionMessage('DROWSY is an unknown status label!');
 
         $service->getHealthReport($fixture);
     }
@@ -135,6 +140,10 @@ class GithubServiceTest extends TestCase
             new Dinosaur('Dennis','Pterodactyl', 7, 'Aviary 1'),
         ];
 
+        yield 'Escaped Dino' => [
+            new Dinosaur('Big Eaty', 'Tyrannosaurus', 15, 'Paddock C'),
+        ];
+
         yield 'Drowsy Dino' => [
             new Dinosaur('Bumpy', 'Triceratops', 10, 'Paddock B'),
         ];
