diff --git a/tests/Unit/Service/GithubServiceTest.php b/tests/Unit/Service/GithubServiceTest.php
index 5653326..c461585 100644
--- a/tests/Unit/Service/GithubServiceTest.php
+++ b/tests/Unit/Service/GithubServiceTest.php
@@ -68,13 +68,9 @@ class GithubServiceTest extends TestCase
         $mockResponse
             ->method('toArray')
             ->willReturn([
-                [
-                    'title' => 'Daisy',
-                    'labels' => [['name' => 'Status: Sick']],
-                ],
                 [
                     'title' => 'Maverick',
-                    'labels' => [['name' => 'Status: Healthy']],
+                    'labels' => [['name' => 'Status: Drowsy']],
                 ],
             ])
         ;
