diff --git a/src/Service/GithubService.php b/src/Service/GithubService.php
index 54365be..e33dfe2 100644
--- a/src/Service/GithubService.php
+++ b/src/Service/GithubService.php
@@ -46,6 +46,11 @@ class GithubService
 
             // Remove the "Status:" and whitespace from the label
             $status = trim(substr($label, strlen('STATUS:')));
+
+            // Determine if we know about the label - throw an exception is we don't
+            if (!in_array($status, [Dinosaur::STATUS_SICK, Dinosaur::STATUS_HUNGRY, Dinosaur::STATUS_HEALTHY])) {
+                throw new \RuntimeException(sprintf('%s is an unknown status label!', $label));
+            }
         }
 
         return $status ?? Dinosaur::STATUS_HEALTHY;
