diff --git a/src/Service/GithubService.php b/src/Service/GithubService.php
index 55cbb0e..220a8bb 100644
--- a/src/Service/GithubService.php
+++ b/src/Service/GithubService.php
@@ -46,6 +46,11 @@ class GithubService
 
             // Remove the "Status:" and whitespace from the label
             $status = trim(substr($label, strlen('Status:')));
+
+            // Determine if we know about the label - throw an exception is we don't
+            if (null === HealthStatus::tryFrom($status)) {
+                throw new \RuntimeException(sprintf('%s is an unknown status label!', $label));
+            }
         }
 
         return HealthStatus::tryFrom($status) ?? HealthStatus::HEALTHY;
