diff --git a/tests/Unit/Service/GithubServiceTest.php b/tests/Unit/Service/GithubServiceTest.php
index b089418..5d2a973 100644
--- a/tests/Unit/Service/GithubServiceTest.php
+++ b/tests/Unit/Service/GithubServiceTest.php
@@ -84,6 +84,8 @@ class GithubServiceTest extends TestCase
 
         $service = new GithubService($mockHttpClient, $mockLogger);
 
+        $this->expectException(\RuntimeException::class);
+
         $service->getHealthReport('Maverick');
     }
 }
