diff --git a/tests/Unit/Service/GithubServiceTest.php b/tests/Unit/Service/GithubServiceTest.php
index c349777..847613a 100644
--- a/tests/Unit/Service/GithubServiceTest.php
+++ b/tests/Unit/Service/GithubServiceTest.php
@@ -85,6 +85,7 @@ class GithubServiceTest extends TestCase
         $service = new GithubService($mockHttpClient, $mockLogger);
 
         $this->expectException(\RuntimeException::class);
+        $this->expectExceptionMessage('Drowsy is an unknown status label!');
 
         $service->getHealthReport('Maverick');
     }
