diff --git a/tests/Unit/Service/GithubServiceTest.php b/tests/Unit/Service/GithubServiceTest.php
index 8fa7ff1..cb77e18 100644
--- a/tests/Unit/Service/GithubServiceTest.php
+++ b/tests/Unit/Service/GithubServiceTest.php
@@ -124,6 +124,8 @@ class GithubServiceTest extends TestCase
 
         $service = new GithubService($mockHttpClient, $this->createMock(LoggerInterface::class));
 
+        $this->expectException(\RuntimeException::class);
+
         $service->getHealthReport($fixture);
     }
 
