diff --git a/src/Service/GithubService.php b/src/Service/GithubService.php
index 45c108c..05fa442 100644
--- a/src/Service/GithubService.php
+++ b/src/Service/GithubService.php
@@ -36,10 +36,25 @@ class GithubService
 
     private function getDinoStatusFromLabels(array $labels): string
     {
-        return str_replace(
-            search: 'STATUS: ',
-            replace: '',
-            subject: strtoupper($labels[0]['name'])
-        );
+        $status = null;
+
+        foreach ($labels as $label) {
+            $label = strtoupper($label['name']);
+
+            // We only care about "Status" labels
+            if (!str_starts_with($label, 'STATUS:')) {
+                continue;
+            }
+
+            // Remove the "Status:" and whitespace from the label
+            $status = trim(substr($label, strlen('STATUS:')));
+
+            // Determine if we know about the label - throw an exception is we don't
+            if (!in_array($status, [Dinosaur::STATUS_SICK, Dinosaur::STATUS_HUNGRY, Dinosaur::STATUS_HEALTHY])) {
+                throw new \RuntimeException(sprintf('%s is an unknown status label!', $label));
+            }
+        }
+
+        return $status ?? Dinosaur::STATUS_HEALTHY;
     }
 }
