diff --git a/tests/Unit/Service/GithubServiceTest.php b/tests/Unit/Service/GithubServiceTest.php
index b9d0a52..c5db1b4 100644
--- a/tests/Unit/Service/GithubServiceTest.php
+++ b/tests/Unit/Service/GithubServiceTest.php
@@ -2,7 +2,6 @@
 
 namespace App\Tests\Unit\Service;
 
-use App\Entity\Dinosaur;
 use App\Service\GithubService;
 use PHPUnit\Framework\TestCase;
 use Psr\Log\LoggerInterface;
@@ -59,7 +58,7 @@ class GithubServiceTest extends TestCase
         ];
     }
 
-    public function testExceptionThrownWithUnknownLabel(string $expectedStatus, string $dinoName): void
+    public function testExceptionThrownWithUnknownLabel(): void
     {
         $mockLogger = $this->createMock(LoggerInterface::class);
         $mockHttpClient = $this->createMock(HttpClientInterface::class);
@@ -88,6 +87,6 @@ class GithubServiceTest extends TestCase
 
         $service = new GithubService($mockHttpClient, $mockLogger);
 
-        self::assertSame($expectedStatus, $service->getHealthReport($dinoName));
+        $service->getHealthReport('Maverick');
     }
 }
