diff --git a/tests/Unit/Service/GithubServiceTest.php b/tests/Unit/Service/GithubServiceTest.php
index 446713a..0c5b748 100644
--- a/tests/Unit/Service/GithubServiceTest.php
+++ b/tests/Unit/Service/GithubServiceTest.php
@@ -30,6 +30,10 @@ class GithubServiceTest extends TestCase
                     'title' => 'Maverick',
                     'labels' => [['name' => 'Status: Healthy']],
                 ],
+                [
+                    'title' => 'Bumpy',
+                    'labels' => [['name' => 'Status: Hungry']],
+                ],
             ])
         ;
 
@@ -56,6 +60,10 @@ class GithubServiceTest extends TestCase
             'HEALTHY',
             'Maverick',
         ];
+        yield 'Hungry Dino' => [
+            'HUNGRY',
+            'Bumpy',
+        ];
     }
 
     public function testExceptionThrownWithUnknownLabel(): void
