diff --git a/tests/Integration/Repository/LockDownRepositoryTest.php b/tests/Integration/Repository/LockDownRepositoryTest.php
index 2ddd131..d12a534 100644
--- a/tests/Integration/Repository/LockDownRepositoryTest.php
+++ b/tests/Integration/Repository/LockDownRepositoryTest.php
@@ -26,10 +26,9 @@ class LockDownRepositoryTest extends KernelTestCase
     {
         self::bootKernel();
 
-        $lockDown = LockDownFactory::createOne([
+        LockDownFactory::createOne([
             'status' => LockDownStatus::ACTIVE,
         ]);
-        dd($lockDown->assertNotPersisted());
 
         $this->assertTrue($this->getLockDownRepository()->isInLockDown());
     }
