diff --git a/tests/Integration/Repository/LockDownRepositoryTest.php b/tests/Integration/Repository/LockDownRepositoryTest.php
index d1df628..d42af0c 100644
--- a/tests/Integration/Repository/LockDownRepositoryTest.php
+++ b/tests/Integration/Repository/LockDownRepositoryTest.php
@@ -43,13 +43,15 @@ class LockDownRepositoryTest extends KernelTestCase
         self::bootKernel();
 
         LockDownFactory::createOne([
-            'status' => LockDownStatus::ACTIVE,
+            'createdAt' => new \DateTimeImmutable('-1 day'),
+            'status' => LockDownStatus::ENDED,
         ]);
         LockDownFactory::createMany(5, [
-            'status' => LockDownStatus::ENDED,
+            'createdAt' => new \DateTimeImmutable('-2 days'),
+            'status' => LockDownStatus::ACTIVE,
         ]);
 
-        $this->assertTrue($this->getLockDownRepository()->isInLockDown());
+        $this->assertFalse($this->getLockDownRepository()->isInLockDown());
     }
 
     private function getLockDownRepository(): LockDownRepository
