diff --git a/tests/Unit/Service/GithubServiceTest.php b/tests/Unit/Service/GithubServiceTest.php
index c42c15a..e0d8432 100644
--- a/tests/Unit/Service/GithubServiceTest.php
+++ b/tests/Unit/Service/GithubServiceTest.php
@@ -17,7 +17,7 @@ class GithubServiceTest extends TestCase
             new Dinosaur('Maverick','Pterodactyl', 7, 'Aviary 1')
         ];
 
-        self::assertSame($expectedDinos,
+        self::assertEquals($expectedDinos,
             $service->updateHealthStatuses([
                 new Dinosaur('Daisy', 'Velociraptor', 2, 'Paddock A'),
                 new Dinosaur('Maverick','Pterodactyl', 7, 'Aviary 1')
