diff --git a/tests/Unit/Service/GithubServiceTest.php b/tests/Unit/Service/GithubServiceTest.php
new file mode 100644
index 0000000..ff21992
--- /dev/null
+++ b/tests/Unit/Service/GithubServiceTest.php
@@ -0,0 +1,32 @@
+<?php
+
+namespace App\Tests\Unit\Service;
+
+use App\Enum\HealthStatus;
+use PHPUnit\Framework\TestCase;
+
+class GithubServiceTest extends TestCase
+{
+    /**
+     * @dataProvider dinoNameProvider
+     */
+    public function testGetHealthReportReturnsCorrectHealthStatusForDino(HealthStatus $expectedStatus, string $dinoName): void
+    {
+        $service = new GithubService();
+
+        self::assertSame($expectedStatus, $service->getHealthReport($dinoName));
+    }
+
+    public function dinoNameProvider(): \Generator
+    {
+        yield 'Sick Dino' => [
+            HealthStatus::SICK,
+            'Daisy',
+        ];
+
+        yield 'Healthy Dino' => [
+            HealthStatus::HEALTHY,
+            'Maverick',
+        ];
+    }
+}
