diff --git a/src/Service/GithubService.php b/src/Service/GithubService.php
index 3a9432c..38462eb 100644
--- a/src/Service/GithubService.php
+++ b/src/Service/GithubService.php
@@ -17,10 +17,19 @@ class GithubService
 
         foreach ($response->toArray() as $issue) {
             if (str_contains($issue['title'], $dinosaur->getName())) {
-                $health = $issue['labels'][0]['name'];
+                $health = $this->getDinoStatusFromLabels($issue['labels']);
             }
         }
 
         return $health;
     }
+
+    private function getDinoStatusFromLabels(array $labels): string
+    {
+        return str_replace(
+            search: 'STATUS: ',
+            replace: '',
+            subject: strtoupper($labels[0]['name'])
+        );
+    }
 }
