diff --git a/tests/Unit/Service/GithubServiceTest.php b/tests/Unit/Service/GithubServiceTest.php
new file mode 100644
index 0000000..ff0a4ba
--- /dev/null
+++ b/tests/Unit/Service/GithubServiceTest.php
@@ -0,0 +1,29 @@
+<?php
+
+namespace App\Tests\Unit\Service;
+
+use App\Entity\Dinosaur;
+use PHPUnit\Framework\TestCase;
+
+class GithubServiceTest extends TestCase
+{
+    public function testDinoHealth(string $expectedStatus, Dinosaur $fixture): void
+    {
+        $service = new GithubService();
+
+        self::assertSame($expectedStatus, $service->getHealthReport($fixture));
+    }
+
+    public function getDino(): \Generator
+    {
+        yield 'Sick Dino' => [
+            'SICK',
+            (new Dinosaur('Daisy', 'Velociraptor'))->setHealth('SICK'),
+        ];
+
+        yield 'Healthy Dino' => [
+            'HEALTHY',
+            new Dinosaur('Maverick','Pterodactyl'),
+        ];
+    }
+}
