diff --git a/src/Service/GithubService.php b/src/Service/GithubService.php
new file mode 100644
index 0000000..1963a0b
--- /dev/null
+++ b/src/Service/GithubService.php
@@ -0,0 +1,17 @@
+<?php
+
+namespace App\Service;
+
+use App\Enum\HealthStatus;
+
+class GithubService
+{
+    public function getHealthReport(string $dinosaurName): HealthStatus
+    {
+        // Call Github API
+
+        // Filter the issues
+
+        return HealthStatus::HEALTHY;
+    }
+}
diff --git a/tests/Unit/Service/GithubServiceTest.php b/tests/Unit/Service/GithubServiceTest.php
index ff21992..b5e5e24 100644
--- a/tests/Unit/Service/GithubServiceTest.php
+++ b/tests/Unit/Service/GithubServiceTest.php
@@ -3,6 +3,7 @@
 namespace App\Tests\Unit\Service;
 
 use App\Enum\HealthStatus;
+use App\Service\GithubService;
 use PHPUnit\Framework\TestCase;
 
 class GithubServiceTest extends TestCase
