diff --git a/src/Service/GithubService.php b/src/Service/GithubService.php
index 04f2b9c..5a19519 100644
--- a/src/Service/GithubService.php
+++ b/src/Service/GithubService.php
@@ -3,17 +3,29 @@
 namespace App\Service;
 
 use App\Entity\Dinosaur;
+use Psr\Log\LoggerInterface;
 use Symfony\Component\HttpClient\HttpClient;
+use Symfony\Contracts\HttpClient\Exception\TransportExceptionInterface;
 
 class GithubService
 {
-    public function getHealthReport(string $dinosaurName): string
+    public function __construct(private LoggerInterface $logger)
+    {
+    }
+
+    public function getHealthReport(string $dinosaurName): ?string
     {
         $health = Dinosaur::STATUS_HEALTHY;
 
         $client = HttpClient::create();
 
-        $response = $client->request('GET', 'https://api.github.com/repos/SymfonyCasts/dino-park/issues');
+        try {
+            $response = $client->request('GET', 'https://api.github.com/repos/SymfonyCasts/dino-park/issues');
+        } catch (TransportExceptionInterface $exception) {
+            $this->logger->info($exception->getMessage());
+
+            return null;
+        }
 
         foreach ($response->toArray() as $issue) {
             if (str_contains($issue['title'], $dinosaurName)) {
