diff --git a/src/Service/GithubService.php b/src/Service/GithubService.php
index 12f1074..7d687fc 100644
--- a/src/Service/GithubService.php
+++ b/src/Service/GithubService.php
@@ -3,15 +3,24 @@
 namespace App\Service;
 
 use App\Entity\Dinosaur;
+use Symfony\Component\HttpClient\HttpClient;
 
 class GithubService
 {
     public function getHealthReport(string $dinosaurName): string
     {
-        // Call Github API
+        $health = Dinosaur::STATUS_HEALTHY;
 
-        // Filter the issues
+        $client = HttpClient::create();
 
-        return Dinosaur::STATUS_HEALTHY;
+        $response = $client->request('GET', 'https://api.github.com/repos/SymfonyCasts/dino-park/issues');
+
+        foreach ($response->toArray() as $issue) {
+            if (str_contains($issue['title'], $dinosaurName)) {
+                $health = $issue['labels'][0]['name'];
+            }
+        }
+
+        return $health;
     }
 }
