diff --git a/src/Service/GithubService.php b/src/Service/GithubService.php
index d337ace..a30cad2 100644
--- a/src/Service/GithubService.php
+++ b/src/Service/GithubService.php
@@ -2,15 +2,25 @@
 
 namespace App\Service;
 
+use Psr\Log\LoggerInterface;
 use Symfony\Component\HttpClient\HttpClient;
+use Symfony\Contracts\HttpClient\Exception\TransportExceptionInterface;
 
 class GithubService
 {
+    public function __construct(private LoggerInterface $logger)
+    {
+    }
+
     public function getHealthReports(array $dinosaurs): array
     {
         $client = HttpClient::create();
 
-        $response = $client->request('GET', 'https://api.github.com/repos/SymfonyCasts/dino-park/issues');
+        try {
+            $response = $client->request('GET', 'https://api.github.com/repos/SymfonyCasts/dino-park/issues');
+        } catch (TransportExceptionInterface $exception) {
+            $this->logger->info($exception->getMessage());
+        }
 
         foreach ($response->toArray() as $issue) {
             foreach($dinosaurs as $dinosaur) {
