diff --git a/src/Service/GithubService.php b/src/Service/GithubService.php
index 1963a0b..2d0b8bc 100644
--- a/src/Service/GithubService.php
+++ b/src/Service/GithubService.php
@@ -3,15 +3,27 @@
 namespace App\Service;
 
 use App\Enum\HealthStatus;
+use Symfony\Component\HttpClient\HttpClient;
 
 class GithubService
 {
     public function getHealthReport(string $dinosaurName): HealthStatus
     {
-        // Call Github API
+        $health = HealthStatus::HEALTHY;
 
-        // Filter the issues
+        $client = HttpClient::create();
 
-        return HealthStatus::HEALTHY;
+        $response = $client->request(
+            method: 'GET',
+            url: 'https://api.github.com/repos/SymfonyCasts/dino-park/issues'
+        );
+
+        foreach ($response->toArray() as $issue) {
+            if (str_contains($issue['title'], $dinosaurName)) {
+
+            }
+        }
+
+        return $health;
     }
 }
