diff --git a/src/Service/GithubService.php b/src/Service/GithubService.php
index 0677466..59c5e66 100644
--- a/src/Service/GithubService.php
+++ b/src/Service/GithubService.php
@@ -16,7 +16,7 @@ class GithubService
         foreach ($response->toArray() as $issue) {
             foreach($dinosaurs as $dinosaur) {
                 if (str_contains($issue['title'], $dinosaur->getName())) {
-                    $dinosaur->setHealth($issue['labels'][0]['name']);
+                    $dinosaur->setHealth($this->getDinoStatusFromLabels($issue['labels']));
                 }
             }
         }
@@ -34,10 +34,19 @@ class GithubService
 
         foreach ($response->toArray() as $issue) {
             if (str_contains($issue['title'], $dinosaur->getName())) {
-                $health = $issue['labels'][0]['name'];
+                $health = $this->getDinoStatusFromLabels($issue['labels']);
             }
         }
 
         return $health;
     }
+
+    private function getDinoStatusFromLabels(array $labels): string
+    {
+        return str_replace(
+            search: 'STATUS: ',
+            replace: '',
+            subject: strtoupper($labels[0]['name'])
+        );
+    }
 }
