diff --git a/src/Service/GithubService.php b/src/Service/GithubService.php
index 3a9432c..8ff32f0 100644
--- a/src/Service/GithubService.php
+++ b/src/Service/GithubService.php
@@ -17,10 +17,29 @@ class GithubService
 
         foreach ($response->toArray() as $issue) {
             if (str_contains($issue['title'], $dinosaur->getName())) {
-                $health = $issue['labels'][0]['name'];
+                $health = $this->getDinoStatusFromLabels($issue['labels']);
             }
         }
 
         return $health;
     }
+
+    private function getDinoStatusFromLabels(array $labels): string
+    {
+        $status = null;
+
+        foreach ($labels as $label) {
+            $label = strtoupper($label['name']);
+
+            // We only care about "Status" labels
+            if (!str_starts_with($label, 'STATUS:')) {
+                continue;
+            }
+
+            // Remove the "Status:" and whitespace from the label
+            $status = trim(substr($label, strlen('STATUS:')));
+        }
+
+        return $status ?? Dinosaur::STATUS_HEALTHY;
+    }
 }
