diff --git a/src/Controller/MainController.php b/src/Controller/MainController.php
index 139bf52..3a9b0ae 100644
--- a/src/Controller/MainController.php
+++ b/src/Controller/MainController.php
@@ -22,7 +22,7 @@ class MainController extends AbstractController
         ];
 
         foreach ($dinos as $dino) {
-            $dino->setHealth($github->getHealthReport($dino));
+            $dino->setHealth($github->getHealthReport($dino) ?? Dinosaur::STATUS_UNKNOWN);
         }
 
         return $this->render('base.html.twig', [
diff --git a/src/Entity/Dinosaur.php b/src/Entity/Dinosaur.php
index e27a39c..00b0aa8 100644
--- a/src/Entity/Dinosaur.php
+++ b/src/Entity/Dinosaur.php
@@ -6,6 +6,7 @@ class Dinosaur
 {
     public const STATUS_HEALTHY = 'HEALTHY';
     public const STATUS_SICK = 'SICK';
+    public const STATUS_UNKNOWN = 'UNKNOWN';
 
     private string $name;
     private string $genus;
