diff --git a/tests/Unit/Service/GithubServiceTest.php b/tests/Unit/Service/GithubServiceTest.php
index 1463b0a..a5b2369 100644
--- a/tests/Unit/Service/GithubServiceTest.php
+++ b/tests/Unit/Service/GithubServiceTest.php
@@ -37,6 +37,7 @@ class GithubServiceTest extends TestCase
         $mockHttpClient
             ->expects(self::once())
             ->method('request')
+            ->with('GET', 'https://api.github.com/repos/SymfonyCasts/dino-park')
             ->willReturn($mockResponse)
         ;
 
