diff --git a/tests/Unit/Service/GithubServiceTest.php b/tests/Unit/Service/GithubServiceTest.php
index ee398a5..5446e88 100644
--- a/tests/Unit/Service/GithubServiceTest.php
+++ b/tests/Unit/Service/GithubServiceTest.php
@@ -32,6 +32,7 @@ class GithubServiceTest extends TestCase
         ;
 
         $mockHttpClient
+            ->expects(self::once())
             ->method('request')
             ->willReturn($mockResponse)
         ;
