diff --git a/src/Controller/MainController.php b/src/Controller/MainController.php
index 1139ee5..751bd78 100644
--- a/src/Controller/MainController.php
+++ b/src/Controller/MainController.php
@@ -3,6 +3,7 @@
 namespace App\Controller;
 
 use App\Entity\Dinosaur;
+use App\Service\GithubService;
 use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
 use Symfony\Component\HttpFoundation\Response;
 use Symfony\Component\Routing\Annotation\Route;
@@ -10,7 +11,7 @@ use Symfony\Component\Routing\Annotation\Route;
 class MainController extends AbstractController
 {
     #[Route(path: '/', name: 'main_controller', methods: ['GET'])]
-    public function index(): Response
+    public function index(GithubService $github): Response
     {
         $dinos = [
             new Dinosaur('Daisy', 'Velociraptor', 2, 'Paddock A'),
@@ -20,6 +21,10 @@ class MainController extends AbstractController
             new Dinosaur('Bumpy', 'Triceratops', 10, 'Paddock B'),
         ];
 
+        foreach ($dinos as $dino) {
+            $dino->setHealth($github->getHealthReport($dino->getName()));
+        }
+
         return $this->render('main/index.html.twig', [
             'dinos' => $dinos,
         ]);
