diff --git a/tests/Unit/Service/GithubServiceTest.php b/tests/Unit/Service/GithubServiceTest.php
index e0dcd91..b7aa757 100644
--- a/tests/Unit/Service/GithubServiceTest.php
+++ b/tests/Unit/Service/GithubServiceTest.php
@@ -35,6 +35,7 @@ class GithubServiceTest extends TestCase
         ;
 
         $mockHttpClient
+            ->expects(self::once())
             ->method('request')
             ->willReturn($mockResponse)
         ;
