diff --git a/tests/Unit/Service/GithubServiceTest.php b/tests/Unit/Service/GithubServiceTest.php
index bc3a3fd..e23b39e 100644
--- a/tests/Unit/Service/GithubServiceTest.php
+++ b/tests/Unit/Service/GithubServiceTest.php
@@ -37,7 +37,7 @@ class GithubServiceTest extends TestCase
         $mockHttpClient
             ->expects(self::once())
             ->method('request')
-            ->with('GET', 'https://api.github.com/repos/SymfonyCasts/dino-park')
+            ->with('GET', 'https://api.github.com/repos/SymfonyCasts/dino-park/issues')
             ->willReturn($mockResponse)
         ;
 
