diff --git a/src/Entity/Dinosaur.php b/src/Entity/Dinosaur.php
index e27a39c..c193ba9 100644
--- a/src/Entity/Dinosaur.php
+++ b/src/Entity/Dinosaur.php
@@ -6,6 +6,7 @@ class Dinosaur
 {
     public const STATUS_HEALTHY = 'HEALTHY';
     public const STATUS_SICK = 'SICK';
+    public const STATUS_HUNGRY = 'HUNGRY';
 
     private string $name;
     private string $genus;
diff --git a/tests/Unit/Entity/DinosaurTest.php b/tests/Unit/Entity/DinosaurTest.php
index 35fe91c..6eef766 100644
--- a/tests/Unit/Entity/DinosaurTest.php
+++ b/tests/Unit/Entity/DinosaurTest.php
@@ -44,10 +44,19 @@ class DinosaurTest extends TestCase
         yield '4 Meter Small Dino' => [4, 'Small'];
     }
 
-    public function testDinoHealth(): void
+    /** @dataProvider getIsAcceptingVisitors */
+    public function testIsAcceptingVisitors(string $health, bool $expectedVisitorStatus): void
     {
         $dino = new Dinosaur('Rex', 'Trex', 0, 'Paddock A');
+        $dino->setHealth($health);
 
-        self::assertTrue($dino->isAcceptingVisitors());
+        self::assertSame($expectedVisitorStatus, $dino->isAcceptingVisitors());
+    }
+
+    public function getIsAcceptingVisitors(): \Generator
+    {
+        yield 'HEALTHY dino is accepting visitors' => [Dinosaur::STATUS_HEALTHY, true];
+        yield 'SICK dino is not accepting visitors' => [Dinosaur::STATUS_SICK, false];
+        yield 'HUNGRY dino is accepting visitors' => [Dinosaur::STATUS_HUNGRY, true];
     }
 }
