diff --git a/src/Entity/Dinosaur.php b/src/Entity/Dinosaur.php
index 53ba658..95d160f 100644
--- a/src/Entity/Dinosaur.php
+++ b/src/Entity/Dinosaur.php
@@ -58,11 +58,7 @@ class Dinosaur
 
     public function isAcceptingVisitors(): bool
     {
-        if ($this->health === self::STATUS_HEALTHY) {
-            return true;
-        }
-
-        return false;
+        return $this->health !== self::STATUS_SICK;
     }
 
     public function setHealth(string $health): self
