diff --git a/src/Enum/HealthStatus.php b/src/Enum/HealthStatus.php
index c5ba116..a57e070 100644
--- a/src/Enum/HealthStatus.php
+++ b/src/Enum/HealthStatus.php
@@ -6,4 +6,5 @@ enum HealthStatus: string
 {
     case HEALTHY = 'Healthy';
     case SICK = 'Sick';
+    case HUNGRY = 'Hungry';
 }
diff --git a/tests/Unit/Entity/DinosaurTest.php b/tests/Unit/Entity/DinosaurTest.php
index e2c5b53..65441b0 100644
--- a/tests/Unit/Entity/DinosaurTest.php
+++ b/tests/Unit/Entity/DinosaurTest.php
@@ -55,4 +55,13 @@ class DinosaurTest extends TestCase
 
         self::assertFalse($dino->isAcceptingVisitors());
     }
+
+    public function testIsAcceptingVisitorsIfHungry(): void
+    {
+        $dino = new Dinosaur('Dennis');
+
+        $dino->setHealth(HealthStatus::HUNGRY);
+
+        self::assertTrue($dino->isAcceptingVisitors());
+    }
 }
