diff --git a/src/Entity/Dinosaur.php b/src/Entity/Dinosaur.php
index d8ccf93..4a1cd77 100644
--- a/src/Entity/Dinosaur.php
+++ b/src/Entity/Dinosaur.php
@@ -55,7 +55,7 @@ class Dinosaur
 
     public function isAcceptingVisitors(): bool
     {
-        return $this->health === HealthStatus::HEALTHY;
+        return $this->health !== HealthStatus::SICK;
     }
 
     public function setHealth(HealthStatus $health): void
