diff --git a/src/Entity/Dinosaur.php b/src/Entity/Dinosaur.php
index 5ec1619..fcddf55 100644
--- a/src/Entity/Dinosaur.php
+++ b/src/Entity/Dinosaur.php
@@ -57,7 +57,7 @@ class Dinosaur
 
     public function isAcceptingVisitors(): bool
     {
-        return $this->health === self::STATUS_HEALTHY;
+        return $this->health !== self::STATUS_HEALTHY;
     }
 
     public function setHealth(string $health): self
