diff --git a/src/Enum/HealthStatus.php b/src/Enum/HealthStatus.php
index c5ba116..a57e070 100644
--- a/src/Enum/HealthStatus.php
+++ b/src/Enum/HealthStatus.php
@@ -6,4 +6,5 @@ enum HealthStatus: string
 {
     case HEALTHY = 'Healthy';
     case SICK = 'Sick';
+    case HUNGRY = 'Hungry';
 }
diff --git a/tests/Unit/Entity/DinosaurTest.php b/tests/Unit/Entity/DinosaurTest.php
index e2c5b53..1eb9621 100644
--- a/tests/Unit/Entity/DinosaurTest.php
+++ b/tests/Unit/Entity/DinosaurTest.php
@@ -55,4 +55,10 @@ class DinosaurTest extends TestCase
 
         self::assertFalse($dino->isAcceptingVisitors());
     }
+
+    public function healthStatusProvider(): \Generator
+    {
+        yield 'Sick dino is not accepting visitors' => [HealthStatus::SICK, false];
+        yield 'Hungry dino is accepting visitors' => [HealthStatus::HUNGRY, true];
+    }
 }
