diff --git a/src/Entity/Dinosaur.php b/src/Entity/Dinosaur.php
index 58e3eb9..5ec1619 100644
--- a/src/Entity/Dinosaur.php
+++ b/src/Entity/Dinosaur.php
@@ -6,6 +6,7 @@ class Dinosaur
 {
     public const STATUS_HEALTHY = 'HEALTHY';
     public const STATUS_SICK = 'SICK';
+    public const STATUS_HUNGRY = 'HUNGRY';
 
     private string $name;
     private string $genus;
diff --git a/tests/Unit/Entity/DinosaurTest.php b/tests/Unit/Entity/DinosaurTest.php
index 2161aa2..bb9102f 100644
--- a/tests/Unit/Entity/DinosaurTest.php
+++ b/tests/Unit/Entity/DinosaurTest.php
@@ -54,4 +54,13 @@ class DinosaurTest extends TestCase
 
         self::assertFalse($dino->isAcceptingVisitors());
     }
+
+    public function testIsAcceptingVisitorsIfHungry(): void
+    {
+        $dino = new Dinosaur('Dennis');
+
+        $dino->setHealth(Dinosaur::STATUS_HUNGRY);
+
+        self::assertTrue($dino->isAcceptingVisitors());
+    }
 }
