diff --git a/tests/Unit/Service/GithubServiceTest.php b/tests/Unit/Service/GithubServiceTest.php
index d8efadc..1da5f2c 100644
--- a/tests/Unit/Service/GithubServiceTest.php
+++ b/tests/Unit/Service/GithubServiceTest.php
@@ -13,7 +13,7 @@ class GithubServiceTest extends TestCase
 {
     private LoggerInterface $mockLogger;
     private MockHttpClient $mockHttpClient;
-    private MockResponse $mockResponse;
+    private ?MockResponse $mockResponse = null;
 
     protected function setUp(): void
     {
@@ -21,6 +21,11 @@ class GithubServiceTest extends TestCase
         $this->mockHttpClient = new MockHttpClient();
     }
 
+    protected function tearDown(): void
+    {
+        $this->mockResponse = null;
+    }
+
     /**
      * @dataProvider dinoNameProvider
      */
