diff --git a/tests/Unit/Service/GithubServiceTest.php b/tests/Unit/Service/GithubServiceTest.php
index c64a413..bafd151 100644
--- a/tests/Unit/Service/GithubServiceTest.php
+++ b/tests/Unit/Service/GithubServiceTest.php
@@ -7,6 +7,7 @@ use App\Service\GithubService;
 use PHPUnit\Framework\TestCase;
 use Psr\Log\LoggerInterface;
 use Symfony\Contracts\HttpClient\HttpClientInterface;
+use Symfony\Contracts\HttpClient\ResponseInterface;
 
 class GithubServiceTest extends TestCase
 {
@@ -17,6 +18,12 @@ class GithubServiceTest extends TestCase
     {
         $mockLogger = $this->createMock(LoggerInterface::class);
         $mockHttpClient = $this->createMock(HttpClientInterface::class);
+        $mockResponse = $this->createMock(ResponseInterface::class);
+
+        $mockHttpClient
+            ->method('request')
+            ->willReturn($mockResponse)
+        ;
 
         $service = new GithubService($mockHttpClient, $mockLogger);
 
