diff --git a/tests/Unit/Service/GithubServiceTest.php b/tests/Unit/Service/GithubServiceTest.php
index d6d1ce9..ee398a5 100644
--- a/tests/Unit/Service/GithubServiceTest.php
+++ b/tests/Unit/Service/GithubServiceTest.php
@@ -19,7 +19,16 @@ class GithubServiceTest extends TestCase
 
         $mockResponse
             ->method('toArray')
-            ->willReturn([])
+            ->willReturn([
+                [
+                    'title' => 'Daisy',
+                    'labels' => [['name' => 'Status: Sick']],
+                ],
+                [
+                    'title' => 'Maverick',
+                    'labels' => [['name' => 'Status: Healthy']],
+                ],
+            ])
         ;
 
         $mockHttpClient
