diff --git a/tests/Unit/Service/GithubServiceTest.php b/tests/Unit/Service/GithubServiceTest.php
index 631f9cb..3dbd366 100644
--- a/tests/Unit/Service/GithubServiceTest.php
+++ b/tests/Unit/Service/GithubServiceTest.php
@@ -20,6 +20,11 @@ class GithubServiceTest extends TestCase
         $mockHttpClient = $this->createMock(HttpClientInterface::class);
         $mockResponse = $this->createMock(ResponseInterface::class);
 
+        $mockResponse
+            ->method('toArray')
+            ->willReturn([])
+        ;
+
         $mockHttpClient
             ->method('request')
             ->willReturn($mockResponse)
