diff --git a/tests/Unit/Entity/DinosaurTest.php b/tests/Unit/Entity/DinosaurTest.php
index 43bbf47..674e79e 100644
--- a/tests/Unit/Entity/DinosaurTest.php
+++ b/tests/Unit/Entity/DinosaurTest.php
@@ -28,4 +28,18 @@ class DinosaurTest extends TestCase
 
         self::assertSame('Large', $dino->getSizeDescription(), 'This is supposed to be a large Dinosaur');
     }
+
+    public function testDinoBetween5And9MetersIsMedium(): void
+    {
+        $dino = new Dinosaur(name: 'Big Eaty', length: 5);
+
+        self::assertSame('Medium', $dino->getSizeDescription(), 'This is supposed to be a medium Dinosaur');
+    }
+
+    public function testDinoUnder5MetersIsSmall(): void
+    {
+        $dino = new Dinosaur(name: 'Big Eaty', length: 4);
+
+        self::assertSame('Small', $dino->getSizeDescription(), 'This is supposed to be a small Dinosaur');
+    }
 }
