diff --git a/tests/Unit/Entity/DinosaurTest.php b/tests/Unit/Entity/DinosaurTest.php
index d2150d8..13a9f08 100644
--- a/tests/Unit/Entity/DinosaurTest.php
+++ b/tests/Unit/Entity/DinosaurTest.php
@@ -21,4 +21,11 @@ class DinosaurTest extends TestCase
         self::assertSame(15, $dino->getLength(), 'Dino length getter is not the same as the constructor');
         self::assertSame('Paddock A', $dino->getEnclosure());
     }
+
+    public function testDinoOver10MetersIsLarge(): void
+    {
+        $dino = new Dinosaur(name: 'Big Eaty', length: 10);
+
+        self::assertSame('Large', $dino->getSizeDescription(), 'This is supposed to be a large Dinosaur');
+    }
 }
