diff --git a/templates/base.html.twig b/templates/base.html.twig
index 9b058a8..437df83 100644
--- a/templates/base.html.twig
+++ b/templates/base.html.twig
@@ -36,7 +36,7 @@
                         <tr>
                             <td>{{ dino.name }}</td>
                             <td>{{ dino.genus }}</td>
-                            <td>{{ dino.length }}</td>
+                            <td>{{ dino.sizeDescription }}</td>
                             <td>{{ dino.enclosure }}</td>
                         </tr>
                     {% endfor %}
