diff --git a/src/Entity/Dinosaur.php b/src/Entity/Dinosaur.php
index 387a3b1..8e92774 100644
--- a/src/Entity/Dinosaur.php
+++ b/src/Entity/Dinosaur.php
@@ -43,12 +43,10 @@ class Dinosaur
             return 'Large';
         }
 
-        if ($this->length < 5) {
-            return 'Small';
-        }
-
-        if ($this->length < 10) {
+        if ($this->length >= 5) {
             return 'Medium';
         }
+
+        return 'Small';
     }
 }
