diff --git a/src/Entity/Dinosaur.php b/src/Entity/Dinosaur.php
index 9f4a4a3..387a3b1 100644
--- a/src/Entity/Dinosaur.php
+++ b/src/Entity/Dinosaur.php
@@ -42,5 +42,13 @@ class Dinosaur
         if ($this->length >= 10) {
             return 'Large';
         }
+
+        if ($this->length < 5) {
+            return 'Small';
+        }
+
+        if ($this->length < 10) {
+            return 'Medium';
+        }
     }
 }
