diff --git a/tests/Unit/Entity/DinosaurTest.php b/tests/Unit/Entity/DinosaurTest.php
index d2150d8..dd9e659 100644
--- a/tests/Unit/Entity/DinosaurTest.php
+++ b/tests/Unit/Entity/DinosaurTest.php
@@ -21,4 +21,15 @@ class DinosaurTest extends TestCase
         self::assertSame(15, $dino->getLength(), 'Dino length getter is not the same as the constructor');
         self::assertSame('Paddock A', $dino->getEnclosure());
     }
+
+    public function testDinoOver15MetersIsLarge(): void
+    {
+        $dino = new Dinosaur(
+            name: 'Big Eaty',
+            genus: 'Tyrannosaurus',
+            length: 15,
+        );
+
+        self::assertSame('Large', $dino->getSpecification(), 'This is supposed to be a large Dinosaur');
+    }
 }
