diff --git a/src/Repository/LockDownRepository.php b/src/Repository/LockDownRepository.php
index df23ef1..4190e0c 100644
--- a/src/Repository/LockDownRepository.php
+++ b/src/Repository/LockDownRepository.php
@@ -22,21 +22,23 @@ class LockDownRepository extends ServiceEntityRepository
         parent::__construct($registry, LockDown::class);
     }
 
-    public function isInLockdown(): bool
+    public function findMostRecent(): ?LockDown
     {
-        // find the most recent lock down
-        $lockDown = $this->createQueryBuilder('lock_down')
+        return $this->createQueryBuilder('lock_down')
             ->orderBy('lock_down.createdAt', 'DESC')
             ->setMaxResults(1)
             ->getQuery()
             ->getOneOrNullResult();
+    }
+
+    public function isInLockdown(): bool
+    {
+        $lockDown = $this->findMostRecent();
 
         if (!$lockDown) {
             return false;
         }
 
-        assert($lockDown instanceof LockDown);
-
         return $lockDown->getStatus() !== LockDownStatus::ENDED;
     }
 }
