diff --git a/src/Service/GithubService.php b/src/Service/GithubService.php
index 5a19519..aab839b 100644
--- a/src/Service/GithubService.php
+++ b/src/Service/GithubService.php
@@ -4,12 +4,12 @@ namespace App\Service;
 
 use App\Entity\Dinosaur;
 use Psr\Log\LoggerInterface;
-use Symfony\Component\HttpClient\HttpClient;
+use Symfony\Contracts\HttpClient\HttpClientInterface;
 use Symfony\Contracts\HttpClient\Exception\TransportExceptionInterface;
 
 class GithubService
 {
-    public function __construct(private LoggerInterface $logger)
+    public function __construct(private HttpClientInterface $httpClient, private LoggerInterface $logger)
     {
     }
 
@@ -17,10 +17,8 @@ class GithubService
     {
         $health = Dinosaur::STATUS_HEALTHY;
 
-        $client = HttpClient::create();
-
         try {
-            $response = $client->request('GET', 'https://api.github.com/repos/SymfonyCasts/dino-park/issues');
+            $response = $this->httpClient->request('GET', 'https://api.github.com/repos/SymfonyCasts/dino-park/issues');
         } catch (TransportExceptionInterface $exception) {
             $this->logger->info($exception->getMessage());
 
diff --git a/tests/Unit/Service/GithubServiceTest.php b/tests/Unit/Service/GithubServiceTest.php
index 2ba6739..7f535ce 100644
--- a/tests/Unit/Service/GithubServiceTest.php
+++ b/tests/Unit/Service/GithubServiceTest.php
@@ -6,6 +6,7 @@ use App\Entity\Dinosaur;
 use App\Service\GithubService;
 use PHPUnit\Framework\TestCase;
 use Psr\Log\LoggerInterface;
+use Symfony\Component\HttpClient\HttpClient;
 
 class GithubServiceTest extends TestCase
 {
@@ -16,7 +17,7 @@ class GithubServiceTest extends TestCase
     {
         $mockLogger = $this->createMock(LoggerInterface::class);
 
-        $service = new GithubService($mockLogger);
+        $service = new GithubService(HttpClient::create(), $mockLogger);
 
         self::assertSame($expectedStatus, $service->getHealthReport($dinoName));
     }
