diff --git a/tests/Unit/Service/GithubServiceTest.php b/tests/Unit/Service/GithubServiceTest.php
index 7f535ce..526be00 100644
--- a/tests/Unit/Service/GithubServiceTest.php
+++ b/tests/Unit/Service/GithubServiceTest.php
@@ -6,7 +6,7 @@ use App\Entity\Dinosaur;
 use App\Service\GithubService;
 use PHPUnit\Framework\TestCase;
 use Psr\Log\LoggerInterface;
-use Symfony\Component\HttpClient\HttpClient;
+use Symfony\Contracts\HttpClient\HttpClientInterface;
 
 class GithubServiceTest extends TestCase
 {
@@ -16,8 +16,9 @@ class GithubServiceTest extends TestCase
     public function testDinoHealth(string $expectedStatus, string $dinoName): void
     {
         $mockLogger = $this->createMock(LoggerInterface::class);
+        $mockHttpClient = $this->createMock(HttpClientInterface::class);
 
-        $service = new GithubService(HttpClient::create(), $mockLogger);
+        $service = new GithubService($mockHttpClient, $mockLogger);
 
         self::assertSame($expectedStatus, $service->getHealthReport($dinoName));
     }
