diff --git a/src/Service/GithubService.php b/src/Service/GithubService.php
index 620484b..55cbb0e 100644
--- a/src/Service/GithubService.php
+++ b/src/Service/GithubService.php
@@ -4,11 +4,11 @@ namespace App\Service;
 
 use App\Enum\HealthStatus;
 use Psr\Log\LoggerInterface;
-use Symfony\Component\HttpClient\HttpClient;
+use Symfony\Contracts\HttpClient\HttpClientInterface;
 
 class GithubService
 {
-    public function __construct(private LoggerInterface $logger)
+    public function __construct(private HttpClientInterface $httpClient, private LoggerInterface $logger)
     {
     }
 
@@ -16,9 +16,7 @@ class GithubService
     {
         $health = HealthStatus::HEALTHY;
 
-        $client = HttpClient::create();
-
-        $response = $client->request('GET', 'https://api.github.com/repos/SymfonyCasts/dino-park/issues');
+        $response = $this->httpClient->request('GET', 'https://api.github.com/repos/SymfonyCasts/dino-park/issues');
 
         $this->logger->info('Request Dino Issues', [
             'dino' => $dinosaurName,
diff --git a/tests/Unit/Service/GithubServiceTest.php b/tests/Unit/Service/GithubServiceTest.php
index b34d199..08d7d66 100644
--- a/tests/Unit/Service/GithubServiceTest.php
+++ b/tests/Unit/Service/GithubServiceTest.php
@@ -6,6 +6,7 @@ use App\Enum\HealthStatus;
 use App\Service\GithubService;
 use PHPUnit\Framework\TestCase;
 use Psr\Log\LoggerInterface;
+use Symfony\Component\HttpClient\HttpClient;
 
 class GithubServiceTest extends TestCase
 {
@@ -16,7 +17,7 @@ class GithubServiceTest extends TestCase
     {
         $mockLogger = $this->createMock(LoggerInterface::class);
 
-        $service = new GithubService($mockLogger);
+        $service = new GithubService(HttpClient::create(), $mockLogger);
 
         self::assertSame($expectedStatus, $service->getHealthReport($dinoName));
     }
